keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -709,6 +732,143 @@ and are not reported here.
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: LocalStorageCapacityIsolationFSQuotaMonitoring
+  - Components depending on the feature gate: kubelet\apiserver\controller-manager\scheduler
+
+###### Does enabling the feature change any default behavior?
+
+None. Behavior will not change.
+When LocalStorageCapacityIsolation is enabled for local ephemeral storage and the backing filesystem for emptyDir volumes supports project quotas and they are enabled, use project quotas to monitor emptyDir volume storage consumption rather than filesystem walk for better performance and accuracy.
+
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. Set the feature gate to false.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Performance changes.
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes, test/e2e_node/quota_lsci_test.go
+
+### Rollout, Upgrade and Rollback Planning
+
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+None.
+
+###### What specific metrics should inform a rollback?
+
+None. User can focus on volume metrics.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Yes.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+LocalStorageCapacityIsolationFSQuotaMonitoring should be turned on only if LocalStorageCapacityIsolation is enabled as well.
+
+### Monitoring Requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+  - A cluster-admin can set kubelet on each node. If the feature gate is disabled, workloads on that node will not use it.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine
+the health of the service?**
+  - Set a quota for the specified volume and try to write to the volume to check if there is a limitation.
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  - N/A.
+
+* **Are there any missing metrics that would be useful to have to improve observability of this feature? **
+  - No.
```

Comment:
I'm not seeing any cluster-admin visibility into the health of this feature.

Reply:
kubelet metrics can show `kubelet_evictions{eviction_signal="ephemeralpodfs.limit"} 1`

I'm not sure if it helps.